Shared/Flows/Onboarding/UserOnboardingViewController.swift

```diff
@@ -102,26 +102,19 @@ class OnboardingMessageTextView: TextView {
         self.isScrollEnabled = false
         self.isSelectable = false
         self.textAlignment = .center
+        self.textContainer.lineBreakMode = .byWordWrapping
+        self.isEditable = false
     }
 
     func set(text: Localized) {
-        let textColor: Color = .textColor
-        let attributedString = AttributedString(text,
-                                                fontType: .smallBold,
-                                                color: textColor)
-
-        self.set(attributed: attributedString,
-                 alignment: .center,
-                 lineCount: 0,
-                 lineBreakMode: .byWordWrapping,
-                 stringCasing: .unchanged,
-                 isEditable: false,
-                 linkColor: .white)
```

Comment:
Here's an example. The textview attributes are set up beforehand so you can just set the text property as expected.